src/faq.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: FAQ
+description: Commonly asked questions and answers.
```

Comment:
Commonly -> Frequently? 

Maybe:

> Frequently asked questions, and answers

Reply:
I'll fix this in my next PR (already underway).